### PR TITLE
Upgrade: dove, move vm version

### PR DESCRIPTION
**Pull request by borispovod**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -116,48 +116,7 @@ Register the parachain:
...
-      "Struct":"MoveStructTag"
-   },
-   "MoveStructTag":{
-      "address":"AccountId",
-      "module":"Text",
-      "name":"Text"
-   }
-}
-```
+Metadata for Polkadot JS can be found in [repository contains types](https://github.com/pontem-network/pontem-types/blob/main/src/index.ts).
```
Comment (cab404) [inline README.md:119]: Maybe change to "Polkadot JS metadata can be found in this repository"?
Or at least "containing types"

Comment (borispovod) [reply README.md]: 👍

Comment (fzzr-) [reply README.md]: By the way, what about passing metadata with ty-mapping directly from pallet? Who already knows?

**Review by fzzr- — APPROVED** [review]
🎉❤️‍🔥

**Pull request merged**